packages/textfield/src/Textfield.ts

```diff
@@ -40,16 +41,19 @@ const textfieldTypes = ['text', 'url', 'tel', 'email', 'password'] as const;
 export type TextfieldType = (typeof textfieldTypes)[number];
 
 /**
+ * @slot default - the visual label for the field
  * @fires input - The value of the element has changed.
  * @fires change - An alteration to the value of the element has been committed by the user.
  */
-export class TextfieldBase extends ManageHelpText(
-    SizedMixin(Focusable, {
-        noDefaultSize: true,
-    })
+export class TextfieldBase extends FieldLabelMixin(
+    ManageHelpText(
+        SizedMixin(Focusable, {
+            noDefaultSize: true,
+        })
+    )
 ) {
     public static override get styles(): CSSResultArray {
-        return [textfieldStyles, checkmarkStyles];
+        return [super.styles || [], textfieldStyles, checkmarkStyles];
```

Comment:
I think if we don'tt spread `super.styles`, it can be problematic? We can end up with a nested array?